utils/validate.js

```diff
@@ -0,0 +1,35 @@
+const YAML = require('yaml')
+const fs = require('fs')
+const path = require('path')
+const Ajv = require('ajv')
+
+function ConfigParse(fileContent, encoding = '') {
```

Comment:
```suggestion
function configParse(fileContent, encoding = '') {
```

Reply:
Please, take a look to the style guide we use to follow: https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers